### ISSUE TITLE
Combination of spread operator and multiple children leads to parser error

### ISSUE THREAD

**Issue by texttechne** [AlexGilleran/jsx-control-statements#46]
Example:
```
<If condition={ true }>
  <span {...{}}>Test</span>
  <div />
</If>
```
Error message:
```
Cannot read property 'name' of undefined
```

**Comment by texttechne** [comment]
Fixed, but not released yet.

**Comment by AlexGilleran** [comment]
Released in v3.1.2 :sunglasses:
